cmd/genesis-nft-miner/src/main.rs

```diff
@@ -21,8 +21,8 @@ use std::time::{Duration, SystemTime, UNIX_EPOCH};
 #[derive(Parser, Debug, Clone)]
 #[clap(version = "0.1.0", author = "Starcoin Core Dev <dev@starcoin.org>")]
 pub struct Options {
-    #[clap(long, default_value = "http://main.seed.starcoin.org")]
-    /// starcoin node http rpc url
+    #[clap(long, default_value = "https://main.seed.starcoin.org")]
+    /// starcoin node https rpc url
```

Comment:
not sure whether this is suitable

Reply:
I use curl visit https://main.seed.starcoin.org no reponse, may be should be http://main.seed.starcoin.org